ReactQt/runtime/src/uimanager.cpp

```diff
@@ -105,9 +105,8 @@ void UIManager::removeChildren(QQuickItem* parent, const QList<int>& removeAtInd
             child->setParentItem(nullptr);
 
             if (unregisterAndDelete) {
-                int childTag = AttachedProperties::get(child)->tag();
-                child->setParent(0);
-                m_views.remove(childTag);
+                child->setParent(nullptr);
```

Comment:
@vkjr should this fix also be applied to ListView based ScrollView there https://github.com/status-im/react-native-desktop/blob/master/ReactQt/runtime/src/componentmanagers/scrollviewmanager.cpp#L109 ?

Reply:
Yes, good point!